### PR TITLE
On-Call schedules end-time and export

### PR DESCRIPTION
**Pull request by Dalje-et**
### What does this PR do? What is the motivation?

Extends documentation for On-Call schedules. In particular, layer end times as well as how to export on-call shifts into 3rd party calendars.

### Merge instructions

Merge readiness:
- [X] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/service_management/on-call/schedules.md [hunk 1]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
 
 Other channel members can select **Take it** to schedule themselves to override your shift.
 
+### Export schedules
+
+The Export Shifts feature allows you to integrate your on-call schedule into your preferred calendar app (for example Google Calendar, Apple Calendar, or Outlook) using a `.webcal` link. Whether you are part of multiple rotations, need to plan personal time around shifts, or want to increase team visibility, this feature ensures your on-call responsibilities are seamlessly accessible and always up-to-date.
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:68]:
```suggestion
The Export Shifts feature allows you to integrate your on-call schedule into your preferred calendar app (for example, Google Calendar, Apple Calendar, or Outlook) using a `.webcal` link. Whether you are part of multiple rotations, planning personal time around shifts, or aiming to increase team visibility, this feature ensures your on-call responsibilities are accessible and always up-to-date.
```

File: content/en/service_management/on-call/schedules.md [hunk 2]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+
+#### How to export and sync you schedules
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:70]:
```suggestion
#### Export and sync schedules
```

File: content/en/service_management/on-call/schedules.md [hunk 3]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+1. **Generate a `.webcal` link**
+    - Go to the [**On-Call** > **Schedules**][1] section in your account.
+    - Select **Export Shifts**. A link is auto-generated for you. Hit **Copy Link** to proceed.
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:73]:
```suggestion
1. To generate a `.webcal` link, go to the [**On-Call** > **Schedules**][1] section in your account.
2. Select **Export My Shifts**. A link is auto-generated for you. 
3. Hit **Copy Link**.
```

File: content/en/service_management/on-call/schedules.md [hunk 4]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+2. After copying the link, use it in your calendar app. For example:
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:74]:
```suggestion
4. After copying the link, use it in your calendar app. For example:
```

File: content/en/service_management/on-call/schedules.md [hunk 5]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+    - Google Calendar: [Follow these instructions from Google (under 'Use a link to add a public calendar').][3]
+    - Outlook: [Microsoft's guide to subscribing to internet calendars.][4]
+    - Apple Calendar: [Learn how to subscribe to calendars on Mac or iPhone.][5]
+
+If your on-call schedule changes, updates are automatically reflected in the linked calendar, ensuring you always have the most accurate view of your responsibilities. If needed, you can revoke access to a previously shared link by generating a new one, which invalidates the old URL.
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:79]:
```suggestion
If your on-call schedule changes, updates are automatically reflected in the linked calendar, ensuring you always have the most accurate view of your responsibilities. If needed, revoke access to a previously shared link by generating a new one, which invalidates the old URL.
```

File: content/en/service_management/on-call/schedules.md [hunk 6]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+
+Most calendar apps support notifications—enable reminders to get alerts before your shifts start, but you can also configure shift reminders through SMS, push notification, and email in your [Datadog On-Call profile settings][6].
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:81]:
```suggestion
Most calendar apps support notifications—enabled reminders to get alerts before your shifts start, but you can also configure shift reminders through SMS, push notification, and email in your [Datadog On-Call profile settings][6].
```

File: content/en/service_management/on-call/schedules.md [hunk 7]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+
+#### Troubleshooting schedule exports
+
+If you experience issues when exporting your On-Call schedule feeds to Google Calendar (for example "could not fetch URL) or Outlook ("Couldn't import calendar. Try again"), test the following fixes when initially subscribing to the calendar via URL:
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:85]:
```suggestion
If you encounter issues when exporting your On-Call schedule feeds to Google Calendar (such as "could not fetch URL) or Outlook ("Couldn't import calendar. Try again"), try the following fixes when initially subscribing to the calendar through URL:
```

File: content/en/service_management/on-call/schedules.md [hunk 8]
```diff
@@ -62,9 +63,37 @@ If you are part of an On-Call rotation, and you know that you will be out of off
...
+
+- Change webcal:// to http:// or https:// at the beginning of the URL.
+  - For example, in place of `webcal://<your_personal_link>`, use `http://<your_personal_link>`.
```
Comment (estherk15) [inline content/en/service_management/on-call/schedules.md:88]:
```suggestion
- Change `webcal://` to `http://` or `https://` at the beginning of the URL. For example, change `webcal://<your_personal_link>` to `http://<your_personal_link>`.
```

**Review by estherk15 — COMMENTED** [review]
Left some suggestions!

**Comment by Dalje-et** [comment]
@estherk15 -- done!

**Review by estherk15 — APPROVED** [review]

**Pull request merged**